webapp/public/style_jury.css

```diff
@@ -48,6 +48,11 @@ tr.summary td { border-top: 1px solid black; }
    clickable in the jury scoreboard and for all team cells */
 .scoreboard_jury td, .scoreboard_jury th { padding: 0; }
 
+/* show pending submissions using a blue corner */
+.score_pending.score_correct             { background: linear-gradient(45deg, #60e760 80%, #6666ff 80%); }
```

Comment:
I personally like 85% better, but I am also fine with 80%

Reply:
I already increased it from [75%](https://github.com/DOMjudge/domjudge/issues/2572#issuecomment-2154180229) to 80%, but I agree that 85% looks even better :smile: Changed! ✅